src/app/components/hero/hero.component.html

```diff
@@ -43,6 +44,9 @@
       }
     </select>
   </div>
+    <button (click)="scrollToTop()" class="scroll" title="top">
+    &#8679;
```

Comment:
we already have a dependency for a library called ng-icons and it has bootstrap icons, so use that
https://ng-icons.github.io/ng-icons/#/browse-icons

```suggestion
    <ng-icon>nameOfTheIcon</ng-icon>
```

i suggest the icon called bootstrapArrowUpCircleFill, but you can go to the website and search the icon you like